CodeHomeWork_2_4/CodeHomeWork_2_4/FileService.cs

```diff
@@ -0,0 +1,64 @@
+﻿namespace CodeHomeWork_2_4
+{
+    public class FileService
+    {
+        private static string _path = string.Empty;
+
+        private static string _fileName = string.Empty;
+
+        static FileService()
+        {
+            _path = Path;
+            _fileName = FileName;
+        }
+
+        public static string Path
+        {
+            get
+            {
+                return _path;
+            }
+            set
+            {
+                _path = value;
+            }
+        }
+
+        public static string FileName
+        {
+            get
+            {
+                return _fileName;
+            }
+            set
+            {
+                _fileName = value;
+            }
+        }
+
+        public void StreamWriter(string log)
+        {
+            DirectoryInfo();
+            using StreamWriter writer = new(_path + _fileName);
```

Comment:
Please use  using StreamWriter writer = new StreamWriter(_path + _fileName); 
that is more general way